scripts/setup-first-time.sh

```
@@ -0,0 +1,195 @@
+MAGENTO_ADMIN_FIRST_NAME=Admin
+MAGENTO_ADMIN_LAST_NAME=User
+MAGENTO_ADMIN_USER=admin
+MAGENTO_ADMIN_PASSWORD=admin123
+MAGENTO_ADMIN_FRONTNAME=admin
+MAGENTO_LOCALE=en_US
+MAGENTO_CURRENCY=USD
+MAGENTO_TIMEZONE=America/New_York
+EOL
+
+# Run the setup script
+cd $MAGENTO_DIR
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)